### PR TITLE
Implemented view for user's rents

### PR DESCRIPTION
**Pull request by ramadis**
## Summary

Implemented view for user's rents

## Test Plan

* Log into your account
* Go to 'My Rents'
* Check your rents

## Trello Card 

 https://trello.com/c/ifC3Jl4F

## Screenshot

my_rents

### CODE REVIEW

File: src/app/components/private/MyRentsController.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    userService.getUser((user) => {
+      bookService.getFromUser(user.objectId, (rents) => {
+        this.rents = rents.results;
+      });
+    });
+
+    this.compareDates = (date) => {
+      const now = new Date();
+      date = new Date(date);
+      return date.getTime() < now.getTime();
```
Comment (JISevilla92) [inline src/app/components/private/MyRentsController.js]: Why don't you just compare the dates without the **.getTime()**?
You could also do this in one line
`
this.compareDates = (date) => date < new Date();
`

File: src/app/services/bookService.js [hunk 2]
```diff
@@ -1,10 +1,61 @@
...
+            'book': {
+              '__type': 'Pointer',
+              'className': 'Book',
+              'objectId': bookId
+            }
+          },
+          'include': 'user'
+        }).then(successGetHistory);
+      },
+      getFromUser: (userId, success, failure) => {
```
Comment (JISevilla92) [inline src/app/services/bookService.js]: Remember to use more descriprive names for the callback functions.. **(successful or success)Callback** or **successfulCb* would be ok.

File: src/app/app.routes.js [hunk 3]
```diff
@@ -117,6 +117,15 @@ angular.module('app').config([
           home: {
             templateUrl: '../app/components/private/rent.html',
             controller: 'RentController',
```
Comment (sbalay) [inline src/app/app.routes.js]: You removed the controllerAs from this state

**Comment by sbalay** [comment]
:+1:

**Pull request merged**